packages/core/contracts/financial-templates/expiring-multiparty/ExpiringMultiPartyCreator.sol

```diff
@@ -111,7 +111,7 @@ contract ExpiringMultiPartyCreator is ContractCreator, Testable, Lockable {
         require(params.expirationTimestamp > block.timestamp, "Invalid expiration time");
         _requireWhitelistedCollateral(params.collateralAddress);
 
-        // We don't want EMP deployers to be able to intentionally or unintentionally set
+        // We don't want EMP deployer's to be able to intentionally or unintentionally set
```

Comment:
nit: I think this was correct originally, since this is just a plural

```suggestion
        // We don't want EMP deployers to be able to intentionally or unintentionally set
```

Reply:
> true!